src/Menu/Menu.spec.js

```diff
@@ -109,4 +110,116 @@ describe('<Menu />', () => {
       assert.strictEqual(list.hasClass('test-class'), true, 'should have the user class');
     });
   });
+
+  describe('mount', () => {
```

Comment:
Read through the tests at `/test/integration/Menu.spec.js` and I don't think there is much overlap. The tests I submitted here check calling of focus methods.

I'm not sure I see the purpose of having separated tests?